.github/workflows/build.yml

```diff
@@ -25,3 +25,5 @@ jobs:
       run: pip install -r requirements.txt
     - name: check-format
       run: python ./check_format.py
+    - name: run-tests
+      run: pytest
```

Comment:
Add a new follow-up command to run `pytest`, which will find all the tests inside the package.